### DISCUSSION TITLE
Pickup and deliver multiple crops

### DISCUSSION THREAD

**Discussion by JustDen1234** [Stephan-S/FS19_AutoDrive#2049] [Ideas]
For automatically feeding animals it would be nice to be able to do the following:

1 Get crop A from a silo
2 Drive to one or more animal pens and deliver the crop
3 Drive to a silo to unload (if there is crop left) the crop
4 Start over with at point 1, but then with crop B, etc. etc.

This should also work if you want to deliver multiple crops from you farms silo to one selling point.

**Comment by cydroid** [comment]
I have the same idea and I think the only thing that would need to be done is to add multiple dump boxes to the interface.  

Have it set up for go next.
